docs/review_process.md

```diff
@@ -0,0 +1,148 @@
+# Review process
+
+Behind the scenes of conan center index, there is a pretty complicated and heavily automated process to review and merge pull requests.
+As there is a lot of activity from many users and various bots (e.g. bumping versions or updating conventions), many PRs are opened every day.
+CCI tries to make process as smooth and simple as possible for the contributors, 
+but it requires complicated logic under the hood, and sometimes the logic might not look that obvious for the newcomers, 
+so this document intends to describe the internal process in detail.
+
+<!-- toc -->
+## Contents
+
+  * [conan-center-bot](#conan-center-bot)
+  * [Green Build](#green-build)
+  * [Unexpected error](#unexpected-error)
+  * [Avoiding Conflicts](#avoiding-conflicts)
+  * [Draft](#draft)
+  * [Getting PR Reviewed](#getting-pr-reviewed)
+  * [Automatic Merges](#automatic-merges)<!-- endToc -->
+  
+## [conan-center-bot](https://github.com/conan-center-bot)
+
+In general, reviews are driven by the automated [bot](https://github.com/conan-center-bot). The bot is responsible for:
+
+- adding or removing labels (such as [No Beta User](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3A%22No+Beta+user%22) or [Docs](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3ADocs))
+- writing comments (most of the time, it's a build status, either failure with logs or success)
+- merging pull requests
+- closing issues (side effect of merging pull requests)
+- starting CI builds
+- assigning CI status (running/failed/successful)
+
+## green build
+
+The first important prerequisite is ensuring your PR is green (build is successful).
+It requires a bit of patience, because there are many PRs are running, and we're building a lot of configurations for a numerous versions of libraries.
+Keep attention to the error messages from the bot, and address all the build failures.
+The bot tries to provide all the helpful information needed to understand and reproduce an issue, such as:
+
+- the profile that failed (in other words, the configuration - architecture, operation system, compiler, etc.)
+- failed command line (it might have failed on early stages - e.g. recipe syntax errors, or later stages, like build or test)
```

Comment:
```suggestion
- Failed command line (it might have failed on early stages, like recipe syntax errors, hook errors or later stages, like build or test).
```